Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -41,3 +41,6 @@
 HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HBHESpikeNoise",12)
 
 CSCHaloData.ExpectedBX = cms.int32(3)
+
+from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrector, ak4PFCHSL1FastL2L3ResidualCorrectorChain
+jetCorrectorsForReco.replace(ak4PFCHSL1FastL2L3CorrectorChain, ak4PFCHSL1FastL2L3ResidualCorrectorChain)
```

Comment:
@davidlange6 these lines complete what have been done in the PR#8817 by @wddgit 
Basically I need to have JEC for CHS jets for data (so including Residual correction) so I replace for data the Corrector Chain sequence to include this correction based on a suggestion from @slava77